websites/W/Wbijam/presence.ts

```diff
@@ -0,0 +1,160 @@
+const presence = new Presence({
+		clientId: "992829484564746290",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let iFrameVideo: boolean,
+	currentTime: number,
+	duration: number,
+	paused: boolean;
+
+interface IFrameData {
+	iframeVideo: {
+		dur: number;
+		iFrameVideo: boolean;
+		paused: boolean;
+		currTime: number;
+	};
+}
+
+presence.on("iFrameData", (data: IFrameData) => {
+	if (data.iframeVideo.dur) {
+		({ iFrameVideo, paused } = data.iframeVideo);
+		currentTime = data.iframeVideo.currTime;
+		duration = data.iframeVideo.dur;
+	}
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/1DbVzt2.png",
+			startTimestamp: browsingTimestamp,
+		},
+		domain = document.location.host,
+		path = document.location.pathname,
```

Comment:
This works the same, just ensure to update references of both `path` to `pathname` and `domain` to `hostname`, unless you had a reason to use `host` instead of `hostname` 
```suggestion
		{ hostname, pathname } = document.location,
```

Reply:
can i use `{ hostname: domain, pathname: path } = document.location`?